rbac/management/utils.py

```diff
@@ -337,3 +337,33 @@ def get_admin_from_proxy(username, request):
 
     is_org_admin = bop_resp.get("data")[index]["is_org_admin"]
     return is_org_admin
+
+
+def api_path_prefix():
+    """Get api path prefix."""
+    path_prefix = os.getenv("API_PATH_PREFIX", "api/")
+    if path_prefix != "":
+        if path_prefix.startswith("/"):
+            path_prefix = path_prefix[1:]
+        if not path_prefix.endswith("/"):
+            path_prefix = path_prefix + "/"
+    return path_prefix
+
+
+def v2response_error_from_errors(errors, exc=None, context=None):
```

Comment:
Can't quite tell but it doesn't look like the "type" field is being added to the error response ever?

Reply:
I will create ticket for it.

Reply:
https://issues.redhat.com/browse/RHCLOUD-34231